Packet++/src/GtpLayer.cpp

```diff
@@ -587,22 +587,25 @@ void GtpV1Layer::parseNextLayer()
 
 	// GTP-U message, try to parse the next layer
 
-	uint8_t subProto = *(uint8_t*)(m_Data + headerLen);
+	uint8_t* payload = m_Data + headerLen;
```

Comment:
this might cause compilation warning on some platforms (casting may be needed here)

Reply:
The casting has been added